docs/filesystems/GoogleCloudStorage.md

```diff
@@ -23,7 +23,6 @@ filesystems {
       #              going forward with the cache hit.
       duplication-strategy = "copy"
     }
-  
```

Comment:
Revert?

Reply:
I thought the extra newline was unintentional, so I removed it. If it was intentional I can revert it.